main/FatesHistoryInterfaceMod.F90

```diff
@@ -5065,27 +5059,23 @@ subroutine update_history_hifrq1(this,nc,nsites,sites,bc_in,bc_out,dt_tstep)
 
          else
 
-            ipa = 0
             site_area_veg_inv = 1._r8/site_area_veg_inv
 
             cpatch => sites(s)%oldest_patch
             do while(associated(cpatch))
 
-               ipa = ipa + 1
-
-
                hio_c_stomata_si(io_si) = hio_c_stomata_si(io_si) + &
                     cpatch%c_stomata * cpatch%total_canopy_area * mol_per_umol * site_area_veg_inv
 
                hio_c_lblayer_si(io_si) = hio_c_lblayer_si(io_si) + &
                     cpatch%c_lblayer * cpatch%total_canopy_area * mol_per_umol * site_area_veg_inv
 
                ! Only accumulate the instantaneous vegetation temperature for vegetated patches
-               if (cpatch%patchno .ne. 0) then
+               !if (cpatch%nocomp_pft_label.ne.nocomp_bareground)then
```

Comment:
Should this be reinstated to avoid potentially adding the bareground patch into `hio_tveg`?

Reply:
yeah, need to clean some things up

Reply:
this is cleaned up now